packages/ketcher-core/src/application/render/restruct/rergroup.js

```diff
@@ -63,19 +63,34 @@ class ReRGroup extends ReObject {
 
     if (ret) ret = ret.extend(BORDER_EXT, BORDER_EXT)
 
+    let attachmentPointsVBox = render.ctab.getAttachmentsPointsVBox(
```

Comment:
If the variable doesn't change, please use `const` instead

Reply:
but I did change the value at line 70